lib/graph3d/Graph3d.js

```diff
@@ -2091,40 +2096,42 @@ Graph3d.prototype._onTouchEnd = function(event) {
  * @param {Event}  event   The event
  */
 Graph3d.prototype._onWheel = function(event) {
-  if (!event) /* For IE. */
-    event = window.event;
-
-  // retrieve delta
-  var delta = 0;
-  if (event.wheelDelta) { /* IE/Opera. */
-    delta = event.wheelDelta/120;
-  } else if (event.detail) { /* Mozilla case. */
-    // In Mozilla, sign of delta is different than in IE.
-    // Also, delta is multiple of 3.
-    delta = -event.detail/3;
-  }
+  if (this.zoomable && (!this.ctrlToZoom || event.ctrlKey)) {
+    if (!event) /* For IE. */
```

Comment:
This line should be above the `if` condition

Reply:
true, I make the change